<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelfTest</class>
 <widget class="QWidget" name="SelfTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>703</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>750</width>
       <height>550</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">*{

font-size:30px;

}


</string>
     </property>
     <property name="title">
      <string>设备自检</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>120</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image:url(:/res/light1_1.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../res/light1_1.png</normaloff>../../res/light1_1.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>120</width>
               <height>120</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>120</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>120</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image:url(:/res/light1_1.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../res/light1_1.png</normaloff>../../res/light1_1.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>120</width>
               <height>120</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>120</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-image:url(:/res/light1_4.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../res/light1_4.png</normaloff>../../res/light1_4.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>120</width>
               <height>120</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="styleSheet">
              <string notr="true">QPushButton{
width:150px;
height:50px;
 background-color: #004080;
  color: white;
  border: none;
  font-weight: bold;
 border-radius: 10px;
}
QPushButton:hover {
	background-color: #0080FF;
}
QPushButton:pressed {
	background-color: #0000FF;
}</string>
             </property>
             <property name="text">
              <string>模块自检</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="styleSheet">
              <string notr="true">QPushButton{
width:150px;
height:50px;
 background-color: #004080;
  color: white;
  border: none;
  font-weight: bold;
 border-radius: 10px;
}
QPushButton:hover {
	background-color: #0080FF;
}
QPushButton:pressed {
	background-color: #0000FF;
}</string>
             </property>
             <property name="text">
              <string>打开风机</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_7">
             <property name="styleSheet">
              <string notr="true">QPushButton{
width:150px;
height:50px;
 background-color: #004080;
  color: white;
  border: none;
  font-weight: bold;
 border-radius: 10px;
}
QPushButton:hover {
	background-color: #0080FF;
}
QPushButton:pressed {
	background-color: #0000FF;
}</string>
             </property>
             <property name="text">
              <string>关闭风机</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>模块状态</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>风机状态</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>温度状态</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
        <zorder>textEdit</zorder>
        <zorder>horizontalLayoutWidget</zorder>
        <zorder>horizontalLayoutWidget_2</zorder>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:url(:/res/abnormal.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../res/abnormal.png</normaloff>../../res/abnormal.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>检测前请先进行模块自检和打开风机！</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
